bsp/genericarm64/genericarm64-standard.scc

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: MIT
```

Comment:
We do have:
```
generic-arm64/generic-arm64.scc
generic-arm64/generic-arm64.cfg
generic-arm64/generic-arm64-standard.scc
```

Let's drop this one from this list, and have it as part of the other clean-up that will follow.

Reply:
Done